From ae1f3c61a839324655ef147e0f10393eeea2c68c Mon Sep 17 00:00:00 2001
From: "Adrien M. BERNEDE" <bernede1@llnl.gov>
Date: Wed, 21 Aug 2019 12:17:54 -0700
Subject: [PATCH] Place user scope config dir in spack root dir

---
 lib/spack/spack/paths.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/spack/spack/paths.py b/lib/spack/spack/paths.py
index 2f7ccf3..24a45b6 100644
--- a/lib/spack/spack/paths.py
+++ b/lib/spack/spack/paths.py
@@ -47,7 +47,7 @@ packages_path      = os.path.join(repos_path, "builtin")
 mock_packages_path = os.path.join(repos_path, "builtin.mock")
 
 #: User configuration location
-user_config_path = os.path.expanduser('~/.spack')
+user_config_path = os.path.join(spack_root, '.spack')
 
 
 opt_path        = os.path.join(prefix, "opt")
-- 
2.8.3

